apps/acrs/sections/summary-data-sections.js

```diff
@@ -185,5 +185,29 @@ module.exports = {
         field: 'partner-why-without-partner'
       }
     ]
+  },
+  'child-details': {
+    steps: [
+      {
+        steps: '/child-details',
+        field: 'child-full-name'
+      },
+      {
+        steps: '/child-details',
+        field: 'child-date-of-birth'
+      },
+      {
+        steps: '/child-details',
+        field: 'child-country'
+      },
+      {
+        steps: '/child-details',
+        field: 'child-living-situation'
+      },
+      {
+        steps: '/child-details',
+        field: 'child-why-without-child'
+      }
+    ]
```

Comment:
If you want to show the aggregated children on the summary pages as I have with parents you can check lines 131-156 above.

You might want to deal with it later though

Reply:
Implemented as indicated, good call.